src/VectorFiber.jl

```diff
@@ -0,0 +1,279 @@
+
+"""
```

Comment:
I do (also here) not fully understand what this is for. The doc string is super short and does not help – What do we need this type for?